wiki/Test-infrastructure.md

```diff
@@ -0,0 +1,73 @@
+Test infrastructure
+---
+
+### Running tests
+
+If you just want to run all available tests please use `./run_tests.py`. This is
+the command that is used by continuous integration. Some useful arguments are
+listed below:
+
+* `--infinite` - keep testing until some error is found.
+* `--non-interactive` - do not run interactive gdb session if tests fail.
+* `--thorough` - generate much more test seeds. Testing will take much more time.
```

Comment:
`--board` parameter is also very important.

Reply:
It needs proper description to be added